system/CodeIgniter.php

```diff
@@ -1065,10 +1009,6 @@ protected function sendResponse()
      */
     protected function callExit($code)
     {
-        // @codeCoverageIgnoreStart
-        exit($code);
-        // @codeCoverageIgnoreEnd
+        exit($code); // @codeCoverageIgnore
```

Comment:
Any chance there is a CS Fixer or Rector rule for converting these start-end for single lines? @samsonasik

Reply:
removing before and after comment (before and next different line) is doable with rector, but I am not sure about placing comment on single line directly right after statement. Probably need to extends the `pComments()` method:

https://github.com/nikic/PHP-Parser/blob/c758510a37218d631fd10f67bca5bccbfef864fa/lib/PhpParser/PrettyPrinterAbstract.php#L449-L457

with some kind of custom attribute to flag single line.